Code/Framework/AzCore/AzCore/IO/IStreamerTypes.h

```diff
@@ -22,9 +22,11 @@ constexpr AZ::u64 operator"" _gib(AZ::u64 value);
 
 namespace AZ::IO::IStreamerTypes
 {
+    using Deadline = AZStd::chrono::microseconds;
+
     // Default value for commonly used priorities.
-    inline constexpr static AZStd::chrono::microseconds s_deadlineNow = AZStd::chrono::microseconds::zero();
-    inline constexpr static AZStd::chrono::microseconds s_noDeadline = AZStd::chrono::microseconds::max();
+    inline constexpr static Deadline s_deadlineNow = AZStd::chrono::microseconds::zero();
+    inline constexpr static Deadline s_noDeadline = AZStd::chrono::microseconds::max();
```

Comment:
These definitions can be changed to `Deadline::zero()` and `Deadline::max()`.

Reply:
Agreed, that's safer. Fixed in the next commit.